packages/webapp/src/elections/api/eden-contract.ts

```diff
@@ -133,14 +130,14 @@ export const getVoteDataRow = async (
 };
 
 const getVoteDataRows = async (
-    opts: VoteDataQueryOptionsByGroup
+    opts: TableQueryOptions
 ): Promise<VoteData[] | undefined> => {
     if (devUseFixtureData)
         return Promise.resolve(
             fixtureVoteDataRows.filter(
                 (vote) =>
-                    vote.index >= opts.lowerBound &&
-                    vote.index <= opts.upperBound
+                    vote.index >= opts.lowerBound! &&
+                    vote.index <= opts.upperBound!
```

Comment:
I would probably do something different here if this weren't purely test data / dev time, but it will never affect production